### PR TITLE
Add support for display envvars

### PR DESCRIPTION
**Pull request by jhermann**
Checks both the PYDEPS_DISPLAY and BROWSER envvars to change the default image file opener.

Also: Extended OS error handling when calling dot, graphviz, and the viewer

### CODE REVIEW

**Comment by jhermann** [comment]
Hmmm, wouldn't this be a great time to drop 2.7 support? 😀

**Comment by thebjorn** [comment]
Nope, still a horrible time to drop 2.7 support :-(

File: pydeps/pydeps.py [hunk 1]
```diff
@@ -43,16 +43,25 @@ def _pydeps(trgt, **kw):
             print(dotsrc)
 
         if not no_output:
-            svg = dot.call_graphviz_dot(dotsrc, fmt)
+            try:
+                svg = dot.call_graphviz_dot(dotsrc, fmt)
+            except OSError as cause:
+                raise RuntimeError("While rendering {!r}: {}".format(output, cause)) from cause
```
Comment (thebjorn) [inline pydeps/pydeps.py:49]: I wish... but I'm still on 2.7, so I'm afraid this is a no-go :-(

**Comment by thebjorn** [comment]
I like your other changes though...

**Comment by jhermann** [comment]
I'll remove the from clauses, then it should work.

**Comment by jhermann** [comment]
Giving up. I have no Py2 on this machine.

**Comment by thebjorn** [comment]
Thanks for all your effort!

**Comment by thebjorn** [comment]
v1.9.7 is out on PyPI.

**Pull request merged**